app/views/pictures/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Pictures</h1>
+
+<% if (@user == current_user) || (@nonprofit.present? && @nonprofit.users.include?(current_user)) %>
```

Comment:
@ianroberts131 lets chat about how to make this line and its purpose be clearer.